src/main/java/com/box/sdk/BoxWebHookSignatureVerifier.java

```diff
@@ -0,0 +1,253 @@
+package com.box.sdk;
```

Comment:
Question:  Does this new BoxWebHooksSignatureVerifier place any addition requirements on the Java SDK such as installing special cryptographic libraries?   Or were these special libraries already required for the  Box Java SDK?   If there are any new requirements they must be documented in the setup instructions for the SDK.